src/pages/index.js

```diff
@@ -100,29 +164,116 @@ function openModal(modal) {
 function closeModal(modal) {
   modal.classList.remove("modal_opened");
   document.removeEventListener("keydown", handleEscKeyPress);
+
+  if (modal === deleteModal) {
+    cardToDelete = null;
+  }
+}
+
+const deleteModalCancelButton = deleteModal.querySelector(
+  ".modal_delete-card_cancel"
+);
+if (deleteModalCancelButton) {
+  deleteModalCancelButton.addEventListener("click", () => {
+    closeModal(deleteModal);
+  });
 }
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = nameInput.value;
-  profileDescription.textContent = descriptionInput.value;
-  closeModal(editModal);
+  setButtonText(profileSubmitButton, true, "Saving...");
+
+  api
+    .editUserInfo({
+      name: nameInput.value,
+      about: descriptionInput.value,
+    })
+    .then((user) => {
+      profileName.textContent = user.name;
+      profileDescription.textContent = user.about;
+      editFormElement.reset();
+      resetValidation(
+        editFormElement,
+        [nameInput, descriptionInput],
+        validationConfig
+      );
+      closeModal(editModal);
+      disableButton(
+        editFormElement.querySelector(".modal__submit-button"),
```

Comment:
`editFormElement.querySelector(".modal__submit-button"),`


You've already found the submit button of the profile form as `profileSubmitButton`


```js
      disableButton(profileSubmitButton, validationConfig);


```

Reply:
This remark hasn't been corrected